prow/cmd/deck/static/dialog-polyfill.css

```diff
@@ -0,0 +1,37 @@
+dialog {
```

Comment:
does this need to be a separate stylesheet?

Reply:
it's from GoogleChrome/dialog-polify. I will add the license to it

Reply:
We should still probably inline this into the prow stylesheet with a comment, this doesn't seem worth another request to load?

Reply:
sure, I will do it tmr